tools/llama-bench/llama-bench.cpp

```diff
@@ -270,6 +396,7 @@ struct cmd_params {
     bool                             no_warmup;
     output_formats                   output_format;
     output_formats                   output_format_stderr;
+    bool                             list_devices;
```

Comment:
I don't think adding this is necessary, just print the device list and exit immediately when seeing `--list-devices` while parsing the command line arguments.